Purchases/Public/RCPurchases.m

```diff
@@ -315,6 +311,16 @@ - (instancetype)initWithAppUserID:(nullable NSString *)appUserID
     return self;
 }
 
+- (void)subscribeToAppStateNotifications {
+    [self.notificationCenter addObserver:self
+                                selector:@selector(applicationDidBecomeActive:)
+                                    name:APP_DID_BECOME_ACTIVE_NOTIFICATION_NAME object:nil];
+    [self.notificationCenter addObserver:self
+                                selector:@selector(syncSubscriberAttributesIfNeeded)
+                                    name:APP_WILL_RESIGN_ACTIVE_NOTIFICATION_NAME
+                                  object:nil];
+}
```

Comment:
unified all `NSNotificationCenter` stuff in a single file, which makes it easier to keep track of subscribed events